app/subscriber/src/components/share-menu/ShareMenu.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { FaEnvelope } from 'react-icons/fa';
+import { toast } from 'react-toastify';
+import { useColleagues, useLookup } from 'store/hooks';
+import {
+  IContentModel,
+  IOptionItem,
+  Modal,
+  OptionItem,
+  Select,
+  Settings,
+  useModal,
+} from 'tno-core';
+
+import * as styled from './styled';
+
+export interface IShareSubMenuProps {
+  /**
+   * The content that has been selected to add to a folder
+   */
+  content: IContentModel[];
+}
+
+/** Component that renders the button that gives users access to a sub menu that will allow them to add selected content to
+ * an existing folder. Or create a new one.
+ */
+export const ShareMenu: React.FC<IShareSubMenuProps> = ({ content }) => {
+  const [options, setOptions] = React.useState<IOptionItem[]>([]);
```

Comment:
Organization convention is as follows.

Hook dependencies first.
React.useState second.
Component variables third.
React.useEffect and functions last.